android/app/src/main/kotlin/com/yubico/authenticator/fido/data/Session.kt

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (C) 2024 Yubico.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.yubico.authenticator.fido.data
+
+import com.yubico.yubikit.fido.ctap.Ctap2Session.InfoData
+import kotlinx.serialization.*
+
+typealias YubiKitFidoSession = com.yubico.yubikit.fido.ctap.Ctap2Session
+
+@Serializable
+data class Options(
+    val clientPin: Boolean,
+    val credMgmt: Boolean,
+    val credentialMgmtPreview: Boolean,
+    val bioEnroll: Boolean?,
+    val alwaysUv: Boolean
+)
+
+fun Map<String, Any?>.getBoolean(
+    key: String,
+    default: Boolean = false
+): Boolean = get(key) as? Boolean ?: default
+
+fun Map<String, Any?>.getOptionalBoolean(
+    key: String
+): Boolean? = get(key) as? Boolean
```

Comment:
Maybe make these extensions on InfoData instead?

Reply:
Will investigate this, makes sense to avoid bloating standard types with extensions.

Reply:
fixed